timezone/src/modules/Timezone.rb

```diff
@@ -1036,16 +1036,13 @@ def Summary
 
     # Checks whether the system has Windows installed
     def system_has_windows?
-      begin
-        win_partitions = disk_analyzer.windows_partitions
-        !win_partitions.empty?
-      rescue NameError => ex
-        # bsc#1058869: Don't enforce y2storage being available
-        log.warn("Caught #{ex}")
-        log.warn("No storage-ng support - not checking for a windows partition")
-        log.warn("Assuming UTC for the hardware clock")
-        false # No windows partition found
-      end
+      disk_analyzer.windows_system?
+    rescue NameError => ex
+      # bsc#1058869: Don't enforce y2storage being available
```

Comment:
Just for confirmation, I guess this is the reason why you are not adding/updating a `yast-storage-ng` dependency in the .spec file, right?

Reply:
Yes, it seems we don't want to have a direct dependency.